src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.html.twig

```diff
@@ -1,27 +1,14 @@
 <!-- START of Symfony Web Debug Toolbar -->
 {% if 'normal' != position %}
     <div id="sfMiniToolbar-{{ token }}" class="sf-minitoolbar" data-no-turbolink>
-        <a href="javascript:void(0);" title="Show Symfony toolbar" tabindex="-1" accesskey="D" onclick="
-            var elem = this.parentNode;
-            if (elem.style.display == 'none') {
-                document.getElementById('sfToolbarMainContent-{{ token }}').style.display = 'none';
-                document.getElementById('sfToolbarClearer-{{ token }}').style.display = 'none';
-                elem.style.display = 'block';
-            } else {
-                document.getElementById('sfToolbarMainContent-{{ token }}').style.display = 'block';
-                document.getElementById('sfToolbarClearer-{{ token }}').style.display = 'block';
-                elem.style.display = 'none'
-            }
-
-            Sfjs.setPreference('toolbar/displayState', 'block');
-        ">
+        <a href="#" title="Show Symfony toolbar" tabindex="-1" id="toolbarMiniToggler" accesskey="D">
```

Comment:
all ids you add should start with ``sf`` (to avoid conflicts with ids of the page) and be suffixed with the token (to allow injecting multiple toolbars, which is a supported feature today)